### PR TITLE
Explaining multiple gesture detectors

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes https://github.com/flutter/website/issues/5595

@goderbauer, please check the verbage. I removed the description of "fighting in the arena", "victory", "defeat", and so on.

### CODE REVIEW

**Comment by goderbauer** [comment]
> I removed the description of "fighting in the arena", "victory", "defeat", and so on.

The doc still appears to use the words "victory" and "defeat" under "Gesture disambiguation". Did you mean to remove those as well?

File: src/ui/advanced/gestures.md [hunk 1]
```diff
@@ -4,8 +4,29 @@ description: How gestures, such as taps and drags, work in Flutter.
 ---
 
 This document explains how to listen for, and respond to,
-_gestures_ in Flutter. Examples of gestures include
-taps, drags, and scaling.
+_gestures_ in Flutter.
+Examples of gestures include taps, drags, and scaling.
+
+You might have multiple gesture recognizers,
```
Comment (goderbauer) [inline src/ui/advanced/gestures.md:10]: I think I would move this new block under "Gesture disambiguation" to explain what the first sentence (about having multiple recognizers) means.

File: src/ui/advanced/gestures.md [hunk 2]
```diff
@@ -153,13 +160,40 @@ many of those widgets already respond to taps or gestures.
...
+* A `ListTile` has a tap recognizer that responds
+  to the entire `ListTile`, and a nested one around
+  a trailing icon button. The screen rect of the
+  trailing icon is now covered by two gesture
+  recognizers that need to negotiate for who handles
+  the gesture if it turns out to be a tap.
+* A single `GestureDetector` covers a screen area
+  configured to handle multiple gestures,
+  such as a long press and a tap.
+  This gesture detector now sends multiple
```
Comment (goderbauer) [inline src/ui/advanced/gestures.md:187]: When reading this sentence I was wondering: WHERE to they send the recognizers TO? I really liked the wording in the previous paragraph about the recognizers negotiating. That's happening here with the long press and tap recognizer as well, maybe re-use that phrasing?

**Review by goderbauer — APPROVED** [review]
LGTM

**Review by atsansone — APPROVED** [review]

File: src/ui/advanced/gestures.md (hunk 2 above)
Comment (atsansone) [reply src/ui/advanced/gestures.md:187]: @sfshaza2 : My LGTM depends on addressing this question. Let us know your thoughts after reviewing this.

Comment (sfshaza2) [reply src/ui/advanced/gestures.md]: @atsansone, I've addressed it.

**Pull request merged**